README.md

```diff
@@ -107,3 +107,25 @@ After installing Rust, enter the `rust` directory and run
 cargo run
 ```
 This will output the available subcommands (which can be viewed as entry points); you can run any of them with e.g. `cargo run mqtt-pub-meta`.
+### Web UI service
```

Comment:
Could we maintain a new line above the title, just for the sake of readability of the markdown itself?